### PR TITLE
feat: implement git-browse-ci

### PR DESCRIPTION
**Pull request by pbnj**
This is a proposal and an implementation to open repo's CI page (e.g. GitHub Actions, GitLab CI, Bitbucket Pipelines) in default browser.

Feedback is welcome, even if feedback is "thanks, but no thanks".

### CODE REVIEW

File: bin/git-browse-ci [hunk 1]
```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
```
Comment (spacewander) [inline bin/git-browse-ci:1]: Let's add `set -e -o pipefail`

Comment (pbnj) [reply bin/git-browse-ci:1]: Done.

File: bin/git-browse-ci [hunk 2]
```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env bash
 
+set -e -o pipefail
```
Comment (spacewander) [inline bin/git-browse-ci:3]: I get 128 exit code when no upstream is given after this modification...

Comment (pbnj) [reply bin/git-browse-ci:3]: It seems to be the combination of `set -e` and `git rev-parse --abbrev-ref HEAD 2&> /dev/null` (specifically the `2&> /dev/null`). 

Changing it to the shorthand version `&> /dev/null` or the longhand version `>/dev/null 2>&1` fixes it for me.

**Review by spacewander — APPROVED** [review]

**Pull request merged**